File: 프로그래머스/lv0/120846. 합성수 찾기/합성수 찾기.js
function solution(n) {
    var answer = 0;
    for(var i=4; i<=n; i++){
        for(var j=2; j<i; j++){
            if(i%j == 0) {
                answer++;
                break;
            }
        }
    }
    return answer;
}

File: 프로그래머스/lv0/120846. 합성수 찾기/합성수 찾기.java
class Solution {
    public int solution(int n) {
        int answer = 0;
        int cnt;
        for(int i=4; i<=n; i++){
            cnt=0;
            for(int j=1; j<=i; j++){
                if(i%j==0) { cnt++; }
                if(cnt>=3){
                    answer++;
                    break;
                }
            }
        }
        return answer;
    }
}